### ISSUE TITLE
Not respecting figure dimensions

### ISSUE THREAD

**Issue by roaldarbol** [executablebooks/mystmd#1274]
## Description
If width is not specified, it seems that images are expanded to fill the width of the page (HTML). Possibly get something like `:autowidth:`  which defaults to true (current behaviour), and if set to false the width is set to the original width of the figure? Or maybe default should be to respect figure size?

## Proposed solution
Having a way to specify that the image should have the actual size would be great.

## Additional notes
You could also set `:width:` for each image, but that seems a lot of work.

Currently on `myst v1.2.5`.

**Comment by parmentelat** [comment]
Id like to +1 this feature
in terms of interface though, I'd suggest we instead provide a special value for `width`: - like e.g. `native` or any other that's not used in CSS - which imho would make things simpler for both devs and authors

or, I like your second suggestion too (default being using the native size) in which case one could get, I guess, the current behaviour with `:width: 100%`

**Comment by roaldarbol** [comment]
Yeah, that's a better idea, I agree! @rowanc1 would it be possible to specify a default in the front matter somehow? Like, if you want all your figures to be 100%, you could specify it in front matter rather than having to specify for each figure?

**Comment by fperez** [comment]
I also see that the only ways to specify width are in `px` or `%`. [Quarto (like LaTeX) also supports regular unit dimensions](https://quarto.org/docs/authoring/figures.html#figure-sizing), which is very useful.  I ended up [reading the code](https://github.com/jupyter-book/mystmd/blob/415ed13813c7867441454ea7e55a7e2d64e0eeb6/packages/myst-to-tex/src/utils.ts#L270) to understand what was possible and what wasn't, as [the docs don't provide too much detail](https://mystmd.org/guide/figures#image-directive) on what attributes and formats are allowed.
